ignite/pkg/cosmosgen/generate.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 const (
-	defaultSdkImport     = "github.com/cosmos/cosmos-sdk"
+	defaultSDKImport     = "github.com/cosmos/cosmos-sdk"
 	moduleCacheNamespace = "generate.setup.module"
+	ibcImport            = "github.com/cosmos/ibc-go/v3"
```

Comment:
Regarding the `v3` suffix I initially though it could be an issue to have a hard coded version because it could lead to bugs during major Cosmos IBC version updates, but then realised that all imports are the same, so any major update should find the hardcoded reference. I could remove the version and handle it dynamically if required.

Reply:
I think the hardcoded version is fine. Updating the IBC versions breaks some network commands as well. It's not so trivial for us to update.